data/hai/hai missions.txt

```diff
@@ -1656,3 +1656,127 @@ mission "Strider notification"
+			choice
+				`	"What's the name of your movie, anyways?"`
+				`	"What's your movie about, anyways?"`
+				`	"I better get going, then."`
+					goto going
+			`	"I don't want to spoil anything, but it's a psychological horror film that explores the nature of isolation, and the shadow that our own mortality casts over our lives." She continues to elaborate on more philosophical themes that seemingly have less and less to do with the actual film, until she notices that your eyes have long since glazed over.`
+			`	"Well, anyways, thanks for your help, Captain. I'll contact you in a month once we're finished filming," says Likeet.`
+				accept
+			label going
+			`	"Thanks for your help, Captain. I'll contact you in a month once we're finished," says Likeet.`
+				accept
+
+
+
+event "hai film crew"
+
+
+
+mission "Hai Film Crew 2"
+	landing
+	name "Hai Film Crew"
+	description "Go to <destination> to pick up the film crew you transported there."
+	source
+		near "Ya Hai" 0 100
+	destination "Darkrest"
+	to offer
+		has "event: hai film crew"
+	on offer
+		dialog
+			`Upon landing your ship, you receive a voice call. Although the sender didn't bother to identify themselves, you soon recognize it as the voice of Likeet, the movie director that you brought to Darkrest a month ago.`
+			`	"Captain <last>! Good to see you again! We've finished filming here and need a ride off this place. I'll see you soon." Before you can say anything, she hangs up.`
+
+
+
+mission "Hai Film Crew 3"
+	landing
+	name "Hai Film Crew"
+	description "Take Likeet's film crew home to their studio on <destination>, so that they can process the film they shot on <origin>. Payment is <payment>."
+	cargo "film equipment" 30
+	passengers 80
+	source "Darkrest"
+	destination
+		attributes hai urban
+	to offer
+		has "Hai Film Crew [2]: done"
+	on offer
+		conversation
+			`The first thing you notice when you land on Darkrest is the mess. It looks as if a tornado has touched down on the set, and you don't remember Likeet talking about natural disasters when she described the movie.`
+			`	"Controlled chaos, I call it," says Likeet, who has somehow materialized beside you. "We're just clearing everything out right now, making sure we didn't forget anything, the usual."`
+			`	It takes a few hours to load everything onto your ship. Once you're finished, the only sign that anyone was here is that the station's surfaces are free of dust.`
+			`	"Alright, captain. We'll need to get to <destination>, so that we can edit the raw footage in our studio. Gonna be nice to finally be back on a planet instead of this tin can." She pauses, as if she's trying to remember a line. "Oh yeah, I'll pay you <payment> once we've arrived. Let's get out of here."`
```

Comment:
```suggestion
			`	"Alright, Captain. We'll need to get to <planet> so that we can edit the raw footage in our studio. Gonna be nice to finally be back on a planet instead of this tin can." She pauses, as if she's trying to remember a line. "Oh yeah, I'll pay you <payment> once we've arrived. Let's get out of here."`
```